templates/static-files.conf

```diff
@@ -0,0 +1,60 @@
+server {
+        listen 80;
+        server_name {{ site_url }};
+
+        {% if use_ssl %}
+        return 301 https://{{ site_url }}$request_uri;
+        {% else %}
+        root {{ site_path }};
+
+        location / {
+                index index.html;
```

Comment:
What about index.htm, index.HTML or index.shtml?  Perhaps it would be better to let the natural index resolution take over here.

Reply:
nginx default is 'index.html'

Reply:
If this is the default is there a particular reason not to trust the default?